lib/semian/circuit_breaker.rb

```diff
@@ -1,13 +1,35 @@
 module Semian
-  class CircuitBreaker
-    attr_reader :state
-
-    def initialize(exceptions:, success_threshold:, error_threshold:, error_timeout:)
+  class CircuitBreaker #:nodoc:
+    def initialize(name, exceptions:, success_threshold:, error_threshold:, error_timeout:, permissions: 0660)
+      @name = "#{name}"
```

Comment:
Why not just `@name = name`? If you want to cast it to String, `name.to_s` will do.